opentelemetry-aws/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.5.1
```

Comment:
Do we need to release new versions of these dependent crates? they should still be compatible with the 0.17.1 without a change right?

Reply:
Yep no user facing API has changed. So probably we don't need release a new version for each crates. I will remove those